LightAPRS-hab/LightAPRS-hab.ino

```diff
@@ -39,16 +64,56 @@ bool alternateSymbolTable = false ; //false = '/' , true = '\'
 char Frequency[9]="144.3900"; //default frequency. 144.3900 for US, 144.8000 for Europe
 
 char comment[50] = "First Testing of Light APRS"; // Max 50 char
-char StatusMessage[50] = "This is a BrightLinks Status Msg"; 
+char StatusMessage[50] = "Status Msg: "; 
 //*****************************************************************************
+// variables for smart_packet 
+Adafruit_Si7021 i2c_tracker = Adafruit_Si7021();
 
+long current_altitude = 0;
+long max_altitude = 0;
 
-unsigned int   BeaconWait=42;  //seconds sleep for next beacon (TX).
+long lastalt = 0; // last updated altitude
+bool balloonPopped = false; // DO NOT CHANGE
+int balloonDescendRepeat = 0; // AGAIN, DO NOT CHANGE
+
+const int numDescendChecks = 5;
+int currentSect = 0;
+
+struct txZones {
+  long secsForTx;
+  long altitude;
+  bool goingUp;
+};
+
+#define NUM_ZONES 8
+struct txZones zones[NUM_ZONES] = {
+  {1, 20000, true},
```

Comment:
all of these have secsForTx set to 1.  let's set some real values.  

how about -> 60, 60, 30, 15, 30, 60, 30, 15 ?

Reply:
Done